openeo-geotrellis/src/main/scala/org/openeo/geotrellis/geotiff/package.scala

```diff
@@ -187,8 +192,23 @@ package object geotiff {
         tileLayout, compression, cellTypes.head, tiffBands, segmentCount, fo,
       )
       (correctedPath, timestamp, croppedExtent, bandIndices)
-    }.collect().toList.asJava
-
+    }.collect().map({
+      case (absolutePath, timestamp, croppedExtent, bandIndices) =>
```

Comment:
```suggestion
      case (absoluteFilePath, timestamp, croppedExtent, bandIndices) =>
```

Rename others as well.